common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1079,6 +1079,10 @@ def get_items(self, course_locator, settings=None, content=None, qualifiers=None
                 For substring matching pass a regex object.
                 For split,
                 you can search by ``edited_by``, ``edited_on`` providing a function testing limits.
+        kwargs:
```

Comment:
looks like you can just include `in_tree_only` as another arg, like above.

Also, follow the same formating :)